components/supersaas/sources/changed-users.js

```diff
@@ -0,0 +1,39 @@
+const supersaas = require('../supersaas.app.js');
+
+module.exports = {
+  key: 'supersaas-changed-users',
+  name: 'New or changed users',
+  description: `This event source tracks new and changed user events and emits them.`,
+  version: '0.0.1',
+
+  props: {
+    supersaas,
+    db: "$.service.db",
+    http: '$.interface.http',
+  },
+
+  hooks: {
+    async activate() {
+      const { $auth } = this.supersaas;
+      const { http } = this;
+
+      this.db.set('activeHooks', await this.supersaas.createHooks([{
+        event: 'M', // change_appointment
+        parent_id: $auth.account,
```

Comment:
Can you avoid passing `parent_id` (and `$auth`) here and simply reference the parent ID within the `createHooks` method?

Reply:
Oh I see in the component above, this won't always be `$auth.account`

Reply:
I don't think so, because the `parent_id` is only `$auth.account` in this particular invocation. See for example in `changed-appointments.js` how `parent_id` is something else entirely.

Reply:
Yep makes sense, thanks